recipe/meta.yaml

```diff
@@ -46,13 +42,12 @@ requirements:
     - pkg-config    # [osx]
   host:
     - python
-    - setuptools
     - cython
     - pybind11
     - pythran
```

Comment:
`pythran>=0.9.12,<0.10.0`
https://github.com/scipy/scipy/blob/v1.7.3/pyproject.toml#L16

Reply:
Yeah, the maintainers bumped `pythran` version because before in `v0.9.11` there was `networkx >=2` with scipy as a circular dependency https://github.com/AnacondaRecipes/pythran-feedstock/blob/2f31968eeff1d8aafe7fc68f6ad6f01aac6317ed/recipe/meta.yaml#L45 and in `v0.9.12` they remove `networkx` completely, 
https://github.com/scipy/scipy/commit/0054fae2bc021bf7a5500c88e4d846a959d8cc32 and https://github.com/scipy/scipy/issues/14335#issuecomment-876976102

But our `network` recipe hasn't already `scipy` as a runtime dependency, https://github.com/AnacondaRecipes/networkx-feedstock/blob/master/recipe/meta.yaml#L24-L31, so we can use `pythran 0.9.11` as well because we haven't v0.9.12 in defaults.